pkgs/top-level/all-packages.nix

```diff
@@ -30461,4 +30461,6 @@ in
   lc3tools = callPackage ../development/tools/lc3tools {};
 
   zktree = callPackage ../applications/misc/zktree {};
+
+  kconf = callPackage ../applications/networking/cluster/kconf { };
```

Comment:
Please sort this entry into a category and there alphabetically.

Reply:
I tried to find a pattern, but there are networking/cluster packages spread across the entire file. Can you point to where this entry should go?

Reply:
There should be a headline like ```### NETWORK`` and yeah we need to programmatically sort this.

Reply:
I couldn't find a `### NETWORK` header, but I moved it near some other Kubernetes-related tools. Is this better?